### PR TITLE
[14.0][IMP] `base_jsonify`: Add `_jsonify_x2m_to_ids` and `_jsonify_format_duration` helpers

### PR DESCRIPTION
**Pull request by ivantodorovich**
IMO these basic helpers (and possible others, as long as they're not opinionated) belong here, as you would need them quite often

### CODE REVIEW

File: base_jsonify/tests/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestJsonifyHelpers(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (simahawk) [inline base_jsonify/tests/test_helpers.py:11]:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Review by simahawk — COMMENTED** [review]
LG, we should probably think about another way of defining/using these converters instead of polluting the base model.

**Review by damdam-s — APPROVED** [review]

**Review by SilvioC2C — APPROVED** [review]

**Comment by ivantodorovich** [comment]
@sbidoul runboat is failing here for no apparent reason: `{"detail":"No log found."}` 😅

**Pull request merged**